babel.config.js

```diff
@@ -12,17 +12,13 @@ module.exports = (api) => {
         '@babel/preset-env',
         {
           targets: {
-            node: 'current',
+            // TODO: can we get this to read from package.json somehow..?
```

Comment:
Can you not read `package.json` and parse the `engines.node` field to find the smallest number?